modules/solid_mechanics/src/materials/SymmIsotropicElasticityTensor.C

```diff
@@ -68,6 +68,15 @@ SymmIsotropicElasticityTensor::shearModulus() const
 }
 
 Real
+SymmIsotropicElasticityTensor::youngsModulus() const
+{
+  if (!_E_set)
+    mooseError("Youngs modulus not set");
+
+  return _E;
+}
```

Comment:
So you are adding a new accessor, is there any way to provide a test case for this? 
(i.e. if it's not tested in the module test suite we can't assure that it won't get changed without proper notification)